libavfilter/vf_drawtext.c

```diff
@@ -940,12 +942,16 @@ static int func_eval_expr_int_format(AVFilterContext *ctx, AVBPrint *bp,
         }
     }
 
+#ifndef __SWITCH__
```

Comment:
Hope we don't forget to retain these next time we do actually upgrade ffmpeg...

-[Unknown]

Reply:
Well, I don't actually foresee that happening. Either way, by that time the Switch free SDK might have implemented these functions.

Reply:
Yeah, that's pretty fair.

-[Unknown]

Reply:
I wouldn't worry about that, also I am here to remind you ;)
Could otherwise probably stub these with some define trickery.
Just as a info, these changes (minus the asm ones) are used in the official toolchain ffmpeg lib too.
By the time ffmpeg will be upgraded, we can probably just use the official lib + some extra codecs.

Reply:
Also just a mention: `__BSD_VISIBLE=1` is what the official build added too, I didn't try it with just the define (original) but I think there might be a specific `== 1` check to that in the newlib port somewhere.